### PR TITLE
[4.0] RavenDB-11421

### PR DESCRIPTION
**Pull request by aviv86**

### CODE REVIEW

File: src/Raven.Client/Documents/LinqExtensions.cs [hunk 1]
```diff
@@ -861,6 +861,20 @@ public static IOrderedQueryable<T> OrderByScore<T>(this IQueryable<T> self)
             return (IOrderedQueryable<T>)queryable;
         }
 
+        /// <summary>
+        /// Perform an initial sort by lucene score.
+        /// </summary>
+        public static IOrderedQueryable<T> ThenByScore<T>(this IQueryable<T> self)
```
Comment (ppekrol) [inline src/Raven.Client/Documents/LinqExtensions.cs]: We need to change this to `IOrderedQueryable` IIRC

File: src/Raven.Client/Documents/LinqExtensions.cs [hunk 2]
```diff
@@ -875,6 +889,20 @@ public static IOrderedQueryable<T> OrderByScoreDescending<T>(this IQueryable<T>
             return (IOrderedQueryable<T>)queryable;
         }
 
+        /// <summary>
+        /// Perform an initial sort by lucene score descending.
+        /// </summary>
+        public static IOrderedQueryable<T> ThenByScoreDescending<T>(this IQueryable<T> self)
```
Comment (ppekrol) [inline src/Raven.Client/Documents/LinqExtensions.cs]: We need to change this to IOrderedQueryable IIRC

**Review by ppekrol — CHANGES_REQUESTED** [review]

File: src/Raven.Client/Documents/LinqExtensions.cs (hunk 1 above)
Comment (ppekrol) [reply src/Raven.Client/Documents/LinqExtensions.cs]: I'm referring to `this IQueryable<T> self`

Comment (aviv86) [reply src/Raven.Client/Documents/LinqExtensions.cs]: fixed

File: src/Raven.Client/Documents/LinqExtensions.cs (hunk 2 above)
Comment (aviv86) [reply src/Raven.Client/Documents/LinqExtensions.cs]: fixed

**Pull request merged**